assets/sass/alert.scss

```diff
@@ -1,6 +1,6 @@
 #rpi-alert-container{
-  position: absolute;
-  width: 30%;
+  position: fixed;
```

Comment:
Revert this commit on the file. The Absolute value was meant to avoid overlapping of alerts. We can also add a class for full width alert but it's a bit too big on desktop resolution. May you raise an issue for responsiveness on this?